qa/standalone/osd/osd-rep-recov-eio.sh

```diff
@@ -119,12 +119,24 @@ function rados_get_data() {
     COUNT=$(ceph pg $pgid query | jq '.info.stats.stat_sum.num_objects_repaired')
     test "$COUNT" = "1" || return 1
 
+    local object_osds=($(get_osds $poolname $objname))
+    local primary=${object_osds[0]}
+    local bad_peer=${object_osds[1]}
     inject_$inject rep data $poolname $objname $dir 0 || return 1
     inject_$inject rep data $poolname $objname $dir 1 || return 1
+    # Force primary to pull from the bad peer, so we can repair it too!
+    set_config osd $primary osd_debug_feed_pullee $bad_peer || return 1
```

Comment:
@dzafman To be more specific:

 Here we eio both the primary and a deliberately chosen peer, and then force primary to pull from that very peer when recovering. Before this change, we'd repair primary only and hence leave that peer staying eio.

Then 5 lines below we eio primary (again) and the other still **good** peer. 
If we do not automatically repair bad peers, then the rados_get call 6 lines below will hang since all copies of that object are broken now!